tokio-console/src/config.rs

```diff
@@ -238,10 +250,49 @@ impl Config {
     }
 
     pub(crate) fn retain_for(&self) -> Option<Duration> {
-        self.retain_for.0
+        self.retain_for.as_ref().and_then(|value| value.0)
+    }
+
+    pub(crate) fn target_addr(&self) -> Uri {
+        self.target_addr
+            .as_ref()
+            .unwrap_or(&default_target_addr())
+            .clone()
+    }
+
+    fn from_path(config_path: ConfigPath) -> color_eyre::Result<Option<Self>> {
+        ConfigFile::from_path(config_path)?
+            .map(|config| config.try_into())
+            .transpose()
+    }
+
+    fn merge_with(self, other: Self) -> Self {
+        Self {
+            target_addr: other.target_addr.or(self.target_addr),
+            env_filter: other.env_filter.or(self.env_filter),
+            retain_for: other.retain_for.or(self.retain_for),
+            view_options: self.view_options.merge_with(other.view_options),
+            subcmd: other.subcmd.or(self.subcmd),
+        }
     }
 }
 
+impl Default for Config {
+    fn default() -> Self {
+        Self {
+            target_addr: Some(default_target_addr()),
+            env_filter: Some(tracing_subscriber::EnvFilter::new("info")),
```

Comment:
I think this should default to `off`, like it did previously when set from the CLI arguments
```suggestion
            env_filter: Some(tracing_subscriber::EnvFilter::new("off")),
```